<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnaPencere</class>
 <widget class="QWidget" name="AnaPencere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnaPencere</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sayi Üret</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tek Sayılar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Çift Sayılar</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetSayilar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetSecilen">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>131</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetSec">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>261</y>
     <width>131</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
